app/src/main/java/com/example/cinemapp/util/MovieUtil.kt

```diff
@@ -26,30 +31,34 @@ object MovieUtil {
         )
     }
 
-    fun map(movieList: List<MovieDTO>): List<MovieCard> = movieList.map { movie -> map(movie) }
+    fun mapListMovie(movieList: List<MovieDTO>): List<MovieCard> =
```

Comment:
Now this is where it becomes messy 😅 this class has a lot of mapping logic, and even though it simple mapping, writing tests for it now would be a chaos. Please create a new class for every further mapping, this can stay now as it is.